test/features/email/extract_email_list_post_test.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+import 'package:tmail_ui_user/features/email/presentation/utils/email_utils.dart';
+
+void main() {
+  group('EmailUtils::extractMailtoLinksFromListPost::', () {
```

Comment:
please add test cases which mail-to have subject also

Reply:
done